src/modules/TextOverlay/info.json

```diff
@@ -30,6 +30,22 @@
                 "Montserrat"
             ]
         },
+        "text_decoration": {
+            "type": "select",
+            "desc": "Select the text decoration style.",
+            "default": "none",
+            "values": [
+                "none",
+                "underline",
+                "overline",
+                "strike"
+            ]
+        },
+        "line_height": {
+            "type": "float",
+            "desc": "width of underline, overline or strike",
+            "default": 0
```

Comment:
Why is the default 0? shouldn't it be something more than 0?

Reply:
@niravasher what do you think about this?

Reply:
Since the default text-decoration is none, let the default line height be 0

Reply:
I can change this to 3 or 4, your call @blurry-x-face . Sorry for silly mistakes

Reply:
change it to 3